src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerCreateForm.tsx

```diff
@@ -64,6 +68,10 @@ export const TemplatesCatalogDrawerCreateForm: FC<TemplatesCatalogDrawerCreateFo
       startVM,
     } = useCreateDrawerForm(namespace, subscriptionData, authorizedSSHKey);
 
+    const vmNameValidated = validateVMNameLength(nameField)
```

Comment:
this part repeat in another component, can be in a util file

Reply:
Fixed!